### PR TITLE
E2E UI Testing // User Signup & Login flows

### PR DESCRIPTION
**Pull request by dhreb**
## Description

- Fix several of our broken e2e tests for user signup & login
- Add error test cases to both user signup & login
- TODO: Delete user endpoint needs to be implemented in UI and called during user signup test

## Affected Dependencies
N/A

## How has this been tested?

- Playwright

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: packages/grid/frontend/tests/e2e/auth/login.spec.ts [hunk 1]
```diff
@@ -9,10 +9,8 @@ test.describe('User Login', () => {
   });
 
   test('should login an existing user', async ({ page }) => {
-    await page.goto('/login');
-
-    await page.getByTestId('email').fill('info@openmined.org');
```
Comment (tcp) [inline packages/grid/frontend/tests/e2e/auth/login.spec.ts:14]: Let's keep the default `info@openmined.org` user

File: packages/grid/frontend/tests/e2e/auth/signup.spec.ts [hunk 2]
```diff
@@ -45,10 +45,14 @@ test.describe('User Sign Up', () => {
   });
 
   test('should successfully register a user', async () => {
+    // NOTE: Until we implement Delete user so that we can clean up created test user accounts
+    // this will ensure a new user is created and the test will pass
+    const testUser = `test-user-${Math.round(Math.random() * 1000)}@gmail.com`;
```
Comment (tcp) [inline packages/grid/frontend/tests/e2e/auth/signup.spec.ts:50]: That's good! Let's expand `testUser` and add the other fields to it.

**Comment by madhavajay** [comment]
@tcp is this mergable and should we do so?

**Comment by IonesioJunior** [comment]
I'm considering closing this PR due to the challenges in updating it after a long period. This will help simplify our PR list and clarify our priorities. What are your thoughts on this? @tcp @dhreb @madhavajay

**Pull request closed without merging**